### DISCUSSION TITLE
pihole blocklist

### DISCUSSION THREAD

**Discussion by bjionek** [MajkiIT/polish-ads-filter#22582] [General] [answered]
Co się dzieje z listami na pihole - są usunięte lub puste, kilka dni temu zniknęła lista [adservers.txt] dziś brak już [ads-and-tracking.txt] ?

**Comment by xxcriticxx** [comment, marked as answer]
@bjionek see https://github.com/MajkiIT/polish-ads-filter/issues/22579
